pages/common/git-send-email.md

```diff
@@ -0,0 +1,25 @@
+# git send-email
+
+> Send a collection of patches as emails.
+> Patches can be specified as files, directions, or a revision list.
+> More information: <https://git-scm.com/docs/git-send-email>.
+
+- Send the last commit in the current branch:
+
+`git send-email -1`
+
+- Send a given commit:
+
+`git send-email -1 {{commit}}`
+
+- Send multiple (e.g. 10) commits in the current branch:
+
+`git send-email -10`
+
+- Send an extra mail before the patch mail(s) (e.g. an introduction to the patch set as a cover letter) and edit the mail(s) to be sent:
+
+`git send-email -{{number of commits} --cover-letter --annotate`
+
+- Send the last commit in the current branch with the patch version (e.g. 2) indicated in the subject header (to appear as [PATCH v2]):
+
+`git send-email -v2 -1`
```

Comment:
Is v2 the option by itself, or is it v? Can you point me to the section in https://git-scm.com/docs/git-send-email where it's mentioned? I am having trouble finding it.

Reply:
Sorry about that, I found the example from https://www.freedesktop.org/wiki/Software/PulseAudio/HowToUseGitSendEmail/ under "Adding Patch Version Information."

Reply:
Ah ok, so the option is `v`. But since it's not there in the git documentation, I am not sure if the freedesktop page uses a modified form of that command somehow. Because we are pointing the git documentation to users, I'll prefer only to include commands that are there in that page.